data/human/war reactions.txt

```diff
@@ -0,0 +1,236 @@
+		conversation
+			`You are relaxing in a cafe while your ship refuels, and a pair of middle-aged captains come to your table. One is Navy, the other is Free Worlds, and they're obviously sisters.`
+			`	"Captain Rain, R.N.S. Camuy," says one. "How's the <ship>, Captain <last>?"`
+			`	"Captain Fiona, F.S. Baobab," says her sister. "We both saw you in battle. You're the war's most accomplished mercenary!`
+			`	"We've been patrolling together and we saw your ship, so we've come to say 'hi.'"`
+			label questions
+			choice
+				`	"Why are Navy and Free Worlds captains traveling together?"`
+					goto navy
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label navy"
+				`	"Have you found any new aliens?"`
+					goto aliens
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label aliens"
+						has "War Reactions: Navy and Free Worlds Captains: label navy"
+				`	"Did you fight on the other side of the war from your sister?"`
+					goto sister
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label sister"
+				`	"Mercenary? I'm not a mercenary."`
+					goto mercenary
+					to display
+						not "War Reactions: Navy and Free Worlds Captains: label mercenary"
+				`	"It was good meeting you both, but I must get back to my business now."`
+					decline
+			label aliens
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label aliens"
+			`	"Only the usual assortment of hoaxes," says Rain. The sisters both laugh at this.`
+			`	"Seriously, though, both the Free Worlds Senate and Republic Parliament are up in arms over fear of aliens," says Fiona. Rain adds, "Only metaphorically up in arms, this time.`
+			`	"If we find new aliens, we'll let the necessary authorities know as soon as possible. You should do the same. Be discrete, avoid conflicts, and seek help if you need it."`
+				goto questions
+			label navy
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label navy"
+			`	"Joint police operation," says Captain Rain. "It's more common than you may think."`
+			`	Captain Fiona nods. "We're shutting down a galactic Spider Funk smuggling operation. It's an illegal hallucinogenic. Not addictive, but still dangerous. The organization running it is huge, well funded, and well armed."`
+			`	"Not a match for a Dreadnought and an Auxiliary, though," brags Rain.`
+			`	"We have to fly all over Republic and Free Worlds space, so we have a joint patrol with one ship from each country."`
+			`	"Of course, we have another mission," says Fiona. "Everywhere we go, we look out for new alien activity."`
+				goto questions
+			label sister
+			action
+				set "War Reactions: Navy and Free Worlds Captains: label sister"
+			`	"Yup," Fiona says. "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
+			`	Rain continues. "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
```

Comment:
```suggestion
			`	"Yup," Fiona says, "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
			`	Rain continues, "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
```
We've changed commas in these situations so many times that you get it wrong in the only places where commas should be stuck!